src/web/components/BackToTop.tsx

```diff
@@ -9,15 +10,15 @@ const iconContainer = css`
     position: relative;
     float: right;
     border-radius: 100%;
-    background-color: ${palette.neutral[100]};
+    background-color: ${background.primary};
     cursor: pointer;
     height: ${iconHeight};
     min-width: ${iconHeight};
 `;
 
 const link = css`
     text-decoration: none;
-    color: ${palette.neutral[100]};
+    color: ${brandText.primary};
```

Comment:
Given this is a link, can we use `brandText.anchorPrimary`?

Reply:
Oooh, nice - shall do